test/acceptance/express-basic-test.js

```diff
@@ -5,7 +5,10 @@ const healthcheck = require("../../lib/health");
 
 test('Basic Healthy Health Check', async t => {
 
-  const server = require('../support/server').server;
+  // avoid problems if this env var is already set from wherever test was run
+  process.env.NODE_HEALTH_ENDPOINT_PATH = "";
```

Comment:
Would be slightly friendlier to hold on to the existing value and restore it on line 36.

Reply:
Process.env set in test, gets cleared-out at the end of the test. Also - these tests are async and terminated early if not passing so line 36 is not guaranteed to be reached

Reply:
> terminated early if not passing

Are failing assertions not caught by the `catch` on line 32? Is `server.close` on L35 not always executed?